### ISSUE TITLE
Need a Guide

### ISSUE THREAD

**Issue by shivpatel333** [matcracker/BedcoreProtect#53]
I want You Guys to make a Guide on How to Get the Logs of the Server which are there in your Plugin.

**Comment by neohunter** [comment]
Yeah, we need examples of how to use the commands, like a getting starting guide
